solutions/swb-app/src/dataSets/createExternalEndpointRequestParser.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import { z } from 'zod';
+
+// eslint-disable-next-line @rushstack/typedef-var
+export const CreateExternalEndpointRequestParser = z.object({
```

Comment:
same as above

Reply:
Good catch. Adding that in with the next PR